buidl-labs/crypto-code-school-inside-tezos
# Academy

### What does this PR do?
1. Implements the whole academy flow.
  - `/tezos/academy/`
  - `/tezos/academy/module-*`
  - `/tezos/academy/module-*/chapter-*`

2. Logic to sync progress both on backend and frontend(localStorage)

### How to test this manually?
1. Go to `/tezos/academy`
2. Make your way through the flow, do a few chapters.
3. Then, log in. This should batch update user's progress on the backend. Either check the console or network tab in devtools to see the api call and response to batch update progress.
4. Do a few more chapters. Progress should sync automatically with the backend when logged in.


### Additional context.
Using the PR URL for the backend right now, as the backend branch is not merged yet. Will update this later on.
